<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgSelectionOptions</class>
 <widget class="QWidget" name="WdgSelectionOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>159</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item column="0" row="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item column="3" row="1">
      <widget class="KoGroupButton" name="intersect">
       <property name="toolTip">
        <string>Intersect</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item column="3" row="0">
      <widget class="KoGroupButton" name="shape">
       <property name="toolTip">
        <string>Vector Selection</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item column="1" row="1">
      <widget class="KoGroupButton" name="replace">
       <property name="toolTip">
        <string>Replace</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item column="6" row="1">
      <widget class="KoGroupButton" name="add">
       <property name="toolTip">
        <string>Add</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item column="0" row="0">
      <widget class="QLabel" name="lblMode">
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="text">
        <string>Mode:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item column="0" row="1">
      <widget class="QLabel" name="lblAction">
       <property name="text">
        <string>Action:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item column="1" row="0">
      <widget class="KoGroupButton" name="pixel">
       <property name="toolTip">
        <string>Pixel Selection</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item column="7" row="1">
      <widget class="KoGroupButton" name="subtract">
       <property name="toolTip">
        <string>Subtract</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item column="8" row="1">
      <widget class="KoGroupButton" name="symmetricdifference">
       <property name="toolTip">
        <string>Symmetric Difference</string>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item column="1" row="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item column="0" row="1">
    <widget class="QCheckBox" name="chkAntiAliasing">
     <property name="text">
      <string>Anti-aliasing</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item colspan="2" column="0" row="3">
    <widget class="KisColorFilterCombo" name="cmbColorLabels"/>
   </item>
   <item colspan="2" column="0" row="2">
    <widget class="QComboBox" name="cmbSampleLayersMode"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KoGroupButton</class>
   <extends>QToolButton</extends>
   <header location="global">KoGroupButton.h</header>
  </customwidget>
  <customwidget>
   <class>KisColorFilterCombo</class>
   <extends>QComboBox</extends>
   <header>kis_color_filter_combo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>